assemblies/installing-odh-v2.adoc

```diff
@@ -22,6 +22,15 @@ Installing Open Data Hub involves the following tasks:
 Version 2 of the Open Data Hub Operator represents an alpha release, accessible only on the *fast* channel. Later releases will change to the *rolling* channel when the Operator is more stable.
 --
 
+[IMPORTANT]
+====
+Data Science Pipelines (DSP) 2.0 contains an installation of Argo Workflows. {productname-short} does not support direct customer usage of this installation of Argo Workflows.
```

Comment:
Just wondering if line 27 is missing context that connects the first and second sentence. In other words, why are the following two statements connected:

1. Data Science Pipelines (DSP) 2.0 contains an installation of Argo Workflows. 
2. {productname-short} does not support direct customer usage of this installation of Argo Workflows.

Do we need some kind of further statement which says why:

Argo workflows is intended to be part of the underlying product architecture, and is not designed to be used separately away from the product usage of ODH/RHOAI.